scripts/eosio_build_centos.sh

```diff
@@ -1,698 +1,312 @@
+fi
+
+
+printf "\\n"
+
+
+printf "Checking MongoDB installation...\\n"
+if [ ! -d $MONGODB_ROOT ]; then
+	printf "Installing MongoDB into ${MONGODB_ROOT}...\\n"
+	curl -OL https://fastdl.mongodb.org/linux/mongodb-linux-x86_64-amazon-$MONGODB_VERSION.tgz \
+	&& tar -xzvf mongodb-linux-x86_64-amazon-$MONGODB_VERSION.tgz \
+	&& mv $SRC_LOCATION/mongodb-linux-x86_64-amazon-$MONGODB_VERSION $MONGODB_ROOT \
+	&& touch $MONGODB_LOG_LOCATION/mongod.log \
+	&& rm -f mongodb-linux-x86_64-amazon-$MONGODB_VERSION.tgz \
+	&& cp -f $CURRENT_DIR/scripts/mongod.conf $MONGODB_CONF \
+	&& mkdir -p $MONGODB_DATA_LOCATION \
+	&& rm -rf $MONGODB_LINK_LOCATION \
+	&& rm -rf $BIN_LOCATION/mongod \
+	&& ln -s $MONGODB_ROOT $MONGODB_LINK_LOCATION \
+	&& ln -s $MONGODB_LINK_LOCATION/bin/mongod $BIN_LOCATION/mongod \
+	|| exit 1
+	printf " - MongoDB successfully installed @ ${MONGODB_ROOT} (Symlinked to ${MONGODB_LINK_LOCATION}).\\n"
+else
+	printf " - MongoDB found with correct version @ ${MONGODB_ROOT} (Symlinked to ${MONGODB_LINK_LOCATION}).\\n"
+fi
+printf "Checking MongoDB C driver installation...\\n"
+if [ ! -d $MONGO_C_DRIVER_ROOT ]; then
+	printf "Installing MongoDB C driver...\\n"
+	curl -LO https://github.com/mongodb/mongo-c-driver/releases/download/$MONGO_C_DRIVER_VERSION/mongo-c-driver-$MONGO_C_DRIVER_VERSION.tar.gz \
+	&& tar -xzvf mongo-c-driver-$MONGO_C_DRIVER_VERSION.tar.gz \
+	&& cd mongo-c-driver-$MONGO_C_DRIVER_VERSION \
+	&& mkdir -p cmake-build \
+	&& cd cmake-build \
+	&& cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=$HOME -DENABLE_BSON=ON -DENABLE_SSL=OPENSSL -DENABLE_AUTOMATIC_INIT_AND_CLEANUP=OFF -DENABLE_STATIC=ON .. \
+	&& make -j"${JOBS}" \
+	&& make install \
+	&& cd ../.. \
+	&& rm mongo-c-driver-$MONGO_C_DRIVER_VERSION.tar.gz \
+	|| exit 1
+	printf " - MongoDB C driver successfully installed @ ${MONGO_C_DRIVER_ROOT}.\\n"
+else
+	printf " - MongoDB C driver found with correct version @ ${MONGO_C_DRIVER_ROOT}.\\n"
+fi
+printf "Checking MongoDB C++ driver installation...\\n"
+if [ ! -d $MONGO_CXX_DRIVER_ROOT ]; then
+	printf "Installing MongoDB C++ driver...\\n"
+	curl -L https://github.com/mongodb/mongo-cxx-driver/archive/r$MONGO_CXX_DRIVER_VERSION.tar.gz -o mongo-cxx-driver-r$MONGO_CXX_DRIVER_VERSION.tar.gz \
+	&& tar -xzvf mongo-cxx-driver-r${MONGO_CXX_DRIVER_VERSION}.tar.gz \
+	&& cd mongo-cxx-driver-r$MONGO_CXX_DRIVER_VERSION/build \
+	&& cmake -DBUILD_SHARED_LIBS=OFF -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=$HOME .. \
+	&& make -j"${JOBS}" VERBOSE=1 \
+	&& make install \
+	&& cd ../.. \
+	&& rm -f mongo-cxx-driver-r$MONGO_CXX_DRIVER_VERSION.tar.gz \
+	|| exit 1
+	printf " - MongoDB C++ driver successfully installed @ ${MONGO_CXX_DRIVER_ROOT}.\\n"
+else
+	printf " - MongoDB C++ driver found with correct version @ ${MONGO_CXX_DRIVER_ROOT}.\\n"
+fi
+
+
+printf "\\n"
+
+
+printf "Checking LLVM with WASM support...\\n"
```

Comment:
The wording here (and a handful of other places) needs to be changed. This is only building LLVM4 libraries. Nothing to do with wasm.